include/kernels/gauge_qcharge.cuh

```diff
@@ -0,0 +1,50 @@
+#include <gauge_field_order.h>
+#include <index_helper.cuh>
+#include <quda_matrix.h>
+#include <su3_project.cuh>
+
+#include <cub/cub.cuh>
```

Comment:
We should not be including cub headers directly, just the cub_helper.cuh should be sufficient.

Reply:
Relic from the original code. Fixed the other instances too.